assets/dev/js/editor/components/template-library/views/template/base.js

```diff
@@ -59,10 +59,14 @@ TemplateLibraryTemplateView = Marionette.ItemView.extend( {
 		};
 	},
 
-	behaviors: {
-		insertTemplate: {
-			behaviorClass: TemplateLibraryInsertTemplateBehavior,
-		},
+	behaviors() {
+		const behaviors = {
+			insertTemplate: {
+				behaviorClass: TemplateLibraryInsertTemplateBehavior,
+			},
+		};
+
+		return elementor.hooks.applyFilters( 'elementor/editor/template-library/preview/behaviors', behaviors, this );
```

Comment:
Here in the `addFilter`

```suggestion
		return elementor.hooks.applyFilters( 'elementor/editor/template-library/template/behaviors', behaviors, this );
```

Reply:
why?